task/01-strings-tasks.js

```diff
@@ -69,7 +69,7 @@ function getStringFromTemplate(firstName, lastName) {
  *   'Hello, Chuck Norris!' => 'Chuck Norris'
  */
 function extractNameFromTemplate(value) {
-    throw new Error('Not implemented');
+    return value.match(/(?<=, ).*[^\!]/g)
```

Comment:
should return string, try to solve without regexp

Reply:
Hello. Should I make changes right here using "Edit file"?